### ISSUE TITLE
No sublime text support

### ISSUE THREAD

**Issue by Johnapolitan** [doki-theme/doki-master-theme#151]

**Comment by Unthrottled** [comment]
Yeah, I looked into this a while back, couldn't figure out if I could add widgets/paint a background image. 

When I get the chance, I'll take a look and see if Sublime Text 4 supports adding background images, or if a plugin can do that.

**Comment by Johnapolitan** [comment]
> Yeah, I looked into this a while back, couldn't figure out if I could add widgets/paint a background image.
> 
> When I get the chance, I'll take a look and see if Sublime Text 4 supports adding background images, or if a plugin can do that.

Okay, thank you!

**Comment by choallin** [comment]
Any update here? :-)

**Comment by bathwaterpizza** [comment]
Just dropping by to say I'd also appreciate a Sublime Text version
